README.md

```diff
@@ -28,3 +28,9 @@ Build hackEDU with us. Here are two ways to get started:
 
 - [Create an issue](https://github.com/hackedu/hack-camp/issues)
 - [Join us on IRC](https://webchat.freenode.net/?channels=hackedu)
+
+## Contributing
+
+If you are ready to contribute please begin by consulting the
```

Comment:
Actually @MaxWofford, thoughts on changing `If you are ready to contribute` to `If you would like to contribute`?